static/temp/foods.json

```diff
@@ -0,0 +1,250 @@
+[
+  {
+    "name": "Banana ripe, slightly ripe, raw",
+    "categoryName": "Fruit and Fruit Juices",
+    "categoryIcon": "EggAltOutlinedIcon",
+    "overview": {
+      "info": "This is a cool food",
+      "NutrientScore": ["A", "B", "C", "D"],
+      "climateImpact": {
+        "title": "Climate Impact",
+        "weight": "20kg co2",
+        "weightTitleSubscript": "Per serving",
+        "equivalent": "200kg CO2 is equivalent to",
+        "data": [
+          {
+            "name": "0.2 gallons of diesel consumed",
+            "icon": "OilBarrelOutlinedIcon"
+          },
+          {
+            "name": "0.2 gallons of diesel consumed",
+            "icon": "DirectionsCarFilledOutlinedIcon"
+          }
+        ]
+      },
+      "usefulTips": [
```

Comment:
1. Rename `usefulTips` to `tips`. 
2. Use relevant example as per the Figma design for data. The design is [here](https://www.figma.com/file/gmopGK9vLw5MATERWlWT9W/Forkfacts-with-material-theme?node-id=2352%3A108323&t=khKMxiMJKoLlM6he-4)

Reply:
done